src/fallbacks.jl

```diff
@@ -139,6 +139,24 @@ function calc_extent(::AbstractFeatureTrait, feature)
 end
 calc_extent(t::AbstractFeatureCollectionTrait, fc) = reduce(Extents.union, filter(!isnothing, collect(extent(f) for f in getfeature(t, fc))))
 
+function _nan_free_extrema(f, points)
+    agg_min = agg_max = f(first(points))
```

Comment:
This is vulnerable to `first(points) == (NaN, NaN)` but I don't imagine that's a scenario that will happen often.  Even so, maybe a check could be good?

Reply:
Ah I guess the tests proved me wrong there? But I'm not sure how that works here...

Reply:
We loop over all the points and the found switch means we use the first non-NaN. These are only used if we never find anything else, in which case NaN is the best we have.

Reply:
Aha, I missed that.  Looks good to me then!